presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -158,6 +163,31 @@ fun EventScreen(
         }
     )
 
+    if(eventScreenState.shouldShowVisitorDialog) {
+        Box(modifier = Modifier.fillMaxSize()) {
+            AddVisitorDialog(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .background(
+                        color = MaterialTheme.colorScheme.backgroundWhiteColor,
+                        shape = RoundedCornerShape(16.dp)
+                    )
+                    .padding(20.dp),
+                email = eventScreenState.visitorEmail,
+                onEmailChanged = { visitorEmail ->
+                    eventScreenEvent(EventScreenEvent.OnVisitorEmailChanged(visitorEmail))
+                },
+                onDialogClose = {
+                    eventScreenEvent(EventScreenEvent.OnShowVisitorDialog(shouldShowVisitorDialog = false))
+                },
+                isValidInput = false,
+                onAddButtonClicked = { visitorEmail ->
+                    eventScreenEvent(EventScreenEvent.CheckVisitorExists(visitorEmail))
+                },
+                isEmailVerifiedSuccess = eventScreenState.isEmailVerifiedSuccess
+            )
+        }
+    }
```

Comment:
Not sure why this is a problem. But when there is an error I display the "Email does not exist". I don't think there is any issue with the AddVisitorDialog as the previews seem to show ok. Could it be something to do with the `Box` As the button is half cut-off

![addvisitor-error](https://user-images.githubusercontent.com/14260802/231195895-304ac702-2a4a-4ece-978e-2ce469a3cf7a.png)

Reply:
Would also be my guess that `AlertDialog` has a fixed height while `Dialog` does not

Reply:
@philipplackner I am using the Dialog. However, I have removed the Box, but still has the same behavior.